packages/wdio-browserstack-service/src/testHub/utils.ts

```diff
@@ -0,0 +1,64 @@
+
+import { BROWSERSTACK_PERCY, BROWSERSTACK_OBSERVABILITY, BROWSERSTACK_ACCESSIBILITY } from '../constants.js'
+import type BrowserStackConfig from '../config.js'
+import { BStackLogger } from '../bstackLogger.js'
+
+export const getProductMap = (config: BrowserStackConfig): any => {
+    return {
+        'observability': config.testObservability.enabled,
+        'accessibility': config.accessibility,
+        'percy': config.percy,
+        'automate': config.automate,
+        'app_automate': config.appAutomate
+    }
+}
+
+export const shouldProcessEventForTesthub = (eventType: string): boolean => {
+    if (process.env[BROWSERSTACK_OBSERVABILITY]) {
+        return true
+    }
+    if (process.env[BROWSERSTACK_ACCESSIBILITY]) {
+        return !(['HookRunStarted', 'HookRunFinished', 'LogCreated'].includes(eventType))
+    }
+    if (process.env[BROWSERSTACK_PERCY] && eventType) {
+        return false
+    }
```

Comment:
process.env.<VALUE> This returns a string, and this if check will always return true.
Either convert to Boolean or use `isTrue` method

Reply:
Done.